02-analytics/technical.md

```diff
@@ -279,7 +242,7 @@ Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-sp
 
 | Data          | Name         | Comments                                      |
 |---------------|--------------|-----------------------------------------------|
-| Event ID      | event        | Hardcoded: `calendar_clicked`                 |
+| Event ID      | event        | Hardcoded: `calendarentry_clicked`                 |
 | Page Name     | pageName     | same ID as for page/screen tracking           |
```

Comment:
Stick to a convention? `pageCode` on section_clicked

Reply:
```suggestion
| Page Code     | pageCode     | same code as for page/screen tracking           |
```